ntbs-service/wwwroot/css/alerts.scss

```diff
@@ -25,15 +25,18 @@
     padding-bottom: 10px;
 }
 
-.transfer-request-information-label {
-    padding-right: 12px;
-}
-
 .transfer-request-information-label--no-wrap {

```

Comment:
If we've removed the default label styling - we could probably just have exist without the modifier.